ui-tests/src/test/java/org/openmrs/reference/AddUserTest.java

```diff
@@ -10,18 +10,21 @@
 
 package org.openmrs.reference;
 
-import org.junit.Ignore;
-import org.openmrs.reference.page.*;
-
 import java.util.List;
 
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.openmrs.reference.groups.BuildTests;
+import org.openmrs.reference.page.*;
```

Comment:
explicitly import classes since this would avail all pages which is not right

Reply:
> explicitly import classes since this would avail all pages which is not right

This is absolutely true!! Have addressed this in the commit with id` 2421397`